rest/clients/jaxrs-cxf/src/main/java/com/connexta/search/rest/jaxrscxf/QueryApi.java

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
❓ Why disable generation only on this package? Why not the other ones?

Reply:
The jaxrs generation had a few issues. Namely, it required some dependencies that weren't being used that made it difficult to use in the ion-source. We haven't had any issues with the spring generation yet, so that can still be generated.

Reply:
Probably worth a comment why the section is commented it.